Dockerfile

```
@@ -251,7 +251,9 @@ FROM djs55/vpnkit@sha256:${VPNKIT_DIGEST} AS vpnkit
 FROM runtime-dev AS dev
 ARG DEBIAN_FRONTEND
 RUN groupadd -r docker
-RUN useradd --create-home --gid docker unprivilegeduser
+RUN useradd --create-home --gid docker unprivilegeduser && \
+  mkdir -p /home/unprivilegeduser/.local/share/docker && \
+  chown -R unprivilegeduser /home/unprivilegeduser
```

Could you use the same style as we use in other places? (`&&` on next line);

```suggestion
RUN useradd --create-home --gid docker unprivilegeduser \
 && mkdir -p /home/unprivilegeduser/.local/share/docker \
 && chown -R unprivilegeduser /home/unprivilegeduser
```

---

fixed